packages/fxa-auth-server/scripts/scan-stripe-metadata.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Stripe } from 'stripe';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+export async function init() {
+  const { stripeHelper } = await setupProcessingTaskObjects(
+    'scan-stripe-metadata'
+  );
+  const stripe: Stripe = stripeHelper.stripe;
```

Comment:
If the type definition is not required because it can be implicitly determined (as in this case), do we want to declare it anyway? I have mixed feelings about this. Arguements for are that it is more explcit. Arugments against is that if you refactor a type name, it entangles alot of more files, and it generates extra import statements. I am good either way, but was also curious about others' opinions.